docker/elg_stance.py

```diff
@@ -188,21 +188,34 @@ def iter_json(text):
 
     JSON documents in the input can be separated from each other
     with whitespace, or nothing at all.
+
+    If the text content does not start with a valid JSON object,
+    then we assume that the whole text is instead the plain text
+    content of a single reply, and synthesize a response
+    accordingly.
     """
 
     p = 0
+    found_json = False
     while True:
         # Skip initial whitespace,
         # because raw_decode _still_ doesn't handle it.
         # https://bugs.python.org/issue15393
 
-        m = re.match(r"\s*", text[p:])
+        m = re.match(r"(?:\s|,|\[|\])*", text[p:])
```

Comment:
This particular change we might want to cherry-pick to the target-aware version, what it does is to allow for the input to be a JSON _array_ as well as a stream of concatenated objects (by ignoring any square brackets and commas around or between items)